lib/models/numero.js

```diff
@@ -186,18 +186,18 @@ async function fetchAll(idVoie) {
 }
 
 async function fetchByToponyme(id) {
-  return mongo.db.collection('numeros').aggregate([
-    {$match: {toponyme: id}},
-    {$lookup: {
-      from: 'voies',
-      let: {idVoie: '$voie'},
-      pipeline: [
-        {$match: {$expr: {$eq: ['$_id', '$$idVoie']}}},
-        {$project: {_id: 1, nom: 1}}
-      ],
-      as: 'voie'
-    }}
-  ]).toArray()
+  const numerosWithToponyme = await mongo.db.collection('numeros').find({toponyme: id}).toArray()
+  const voies = await mongo.db.collection('voies').find({_id: {$in: numerosWithToponyme.map(r => r.voie)}}).toArray()
+
+  for (const numero of numerosWithToponyme) {
```

Comment:
Un map serait plus adapté plutôt que modifier l'objet `numero` dans un for of.
L'immutabilité des objets doit toujours être préférée quand c'est possible